src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -299,9 +299,11 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
    */
   private isValidDateRange (minDate: Date, maxDate: Date, dateStrToValidate: string): boolean {
     if (!dateStrToValidate) { return true }
-    // only compare year/month/day (ignore time)
-    const date = new Date(dateStrToValidate) // Input is in the format of MM dd, yyyy
-    return (date >= minDate && date <= maxDate)
+    // Input is in the format of MM dd, yyyy - only compare year/month/day (ignore time)
+    const utcDateStr = new Date(dateStrToValidate + ' 00:00 UTC').toISOString().split('T')[0]
+    const pstDate = this.yyyyMmDdToDate(utcDateStr)
+    console.log(pstDate)
```

Comment:
Does `utcDateStr` in UTC have a time of 00:00 regardless of your workstation's time and time offset?

Does `pstDate` in UTC have a time of 00:00 ?

If so then I believe this will work correctly.

(Please remove console.log.)

Reply:
1. Yes
2.   yyyyMmDdToDate (dateStr: string): Date {
 -  `Converts a date string (YYYY-MM-DD) to a Date object at 12:00:00 am Pacific time.`
 - `   * @example 2021-11-22 -> 2021-11-22T08:00:00.00Z`

Reply:
Then this doesn't work, since you're comparing utcDateStr with time 00:00 with pstDate with time 08:00.

Reply:
This is alot of reinventing the wheel, when a library could easily take care of most of this. 

Luckily for us, I think we're mostly using PST, even though there are technically 3 timezones in BC:

PDT, MST, MDT

Reply:
Did you look at using `createUtcDate()` (in Date Mixin) for creating a Date to compare?

Reply:
No, I just want to get this PR merged

Reply:
I got confused there for a sec.

So `pstDate` is UTC (with Pacific offset) and so are `minDate` and maxDate` ? Then this is fine.

The business decided that displayed dates/times would be in Pacific timezone (PST/PDT) so that's all we have to worry about.